src/S3/MultipartUploader.php

```diff
@@ -59,6 +59,7 @@ public function __construct(
         parent::__construct($client, $source, array_change_key_case($config) + [
             'bucket' => null,
             'key'    => null,
+            'exception_class' => 'Aws\S3\Exception\S3MultipartUploadException'
```

Comment:
If you import the exception with a `use` statement, `::class` should just expand the classname into a fully-qualified one. [Example](https://3v4l.org/LiANN)